main.go

```diff
@@ -46,6 +46,11 @@ func main() {
 			Usage:  "slack template",
 			EnvVar: "PLUGIN_TEMPLATE",
 		},
+		cli.StringFlag{
+			Name:   "fallback",
+			Usage:  "slack fallback",
+			EnvVar: "PLUGIN_FALLBACK",
+		},
```

Comment:
you need to get the value and pass to the configuration object here: https://github.com/drone-plugins/drone-slack/blob/6bd3a07ba87000bc85932641fb497f9bf377f2a0/main.go#L220:L229